common/display/displayqueue.cpp

```diff
@@ -789,4 +789,11 @@ void DisplayQueue::HandleIdleCase() {
   idle_tracker_.idle_lock_.unlock();
 }
 
+bool DisplayQueue::SetActiveConfig(drmModeModeInfo& mode_info) {
+  // update the Acive Mode
+  mode_ = mode_info;
+
+  return SetPowerMode(kOn);
```

Comment:
okay. But Long back when tried It didn't work. With latest code base it is working even when removed.
I'll include this change also. Thanks for the comment.